### DISCUSSION TITLE
Stabillity

### DISCUSSION THREAD

**Discussion by slesinger** [frntc/Sidekick64#47] [Q&A] [answered]
I have strange stability issues show on multiple places.
- Pressing Shift+F8 will take me to Basic screen which works normally but it returns to SK64 menu arbitrarily after several seconds.
- Staying in the SK64 menu freshes screen every now and then.
- Selecting Final Cartridge in freezer menu will show blue screen only. (I am not sure but I think it was working for some time when I have switched on SK64 for the first time.)
- Nordic Power cartridge works until I want to quit to Power basic.

This is not a complete list, just few examples. It is frustrating.

I have ordered SK64 from jlcpcb.com. That is why I have build 5 of them :-) I tried another one with the same result.

I have RPI 3B v1.2. I power it from the C64, A13 is in horizontal position and multiplexer is on d=0. SD card is Kingston 32GB.

Do you have any ideas what could be wrong here? Thanks

**Comment by slesinger** [comment]
in addition: I have tried another SD card and also to power SK64 from USB power adapter. No luck.
I cannot find information on meaning of the 4 LEDs. I wonder if it can help to debug.

**Reply by frntc** [reply]
You are using a Raspberry Pi 3B and not 3B+? Then this is the problem. Which SoC (exact designation) is on your RPi?

**Reply by slesinger** [reply]
The board says exactly:
```
Raspberry Pi 3 Model B V1.2
(C) Raspberry Pi 2015
Made in the UK
FCC ID: 2A8CB-RPI32
IC: 20953-RPI32
KCE MC1 1709
94V-0 F1 C4
```
CPU is:
```
BROADCOM
BCM2837RIFBG
TN1712 P11
475-06 N3 W
```

**Comment by frntc** [comment, marked as answer]
The SoC of the RPi 3A+/B+ is a BCM2837B0. The problems you're experiencing are timing issues as the 3B behaves a bit more slowly than the 3B. There might be a chance to get it running ok-ish when you overclock the RPi and play with the bus timings, but I've never spent time of figuring these out (as said, the 3A+/B+ is the RPi of choice and supported halfway stable alternatives results in too many combinations to support :-/)

**Reply by slesinger** [reply]
I have overclocked ```arm_freq=1350``` so far and it seems to be stable now. CPU temp is reported <55C after 30 minutes of operation.

For value of ```1300``` I still had intermittent issues booting FC3. Thank you.

**Reply by frntc** [reply]
Seems you're lucky, if this is the only value that needs adjustment! :-) In case it becomes unstable, keep in mind that some more tuning might be necessary.

(I never found that heat is a problem in my overclocking experiments, maybe because I use only one CPU core)
